testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -362,5 +364,8 @@
 MMTF = resource_filename(__name__, 'data/173D.mmtf')
 MMTF_gz = resource_filename(__name__, 'data/5KIH.mmtf.gz')
 
+ALIGN_BOUND = resource_filename(__name__, 'data/analysis/align_bound.pdb.gz')
+ALIGN_UNBOUND = resource_filename(__name__, 'data/analysis/align_unbound.pdb.gz')
```

Comment:
Are there any other two protein datafiles we already have and could use for this test? I didn't find any skipping over the list.